test/prow/cluster_create_test_setup.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+ASSISTED_SERVICE_URL="https://api.stage.openshift.com/api/assisted-install/v2"
+
+PULL_SECRET=$(curl -sSf -X POST -H "Content-Type: application/json" -H "Authorization: Bearer ${OCM_TOKEN}" \
+              https://api.stage.openshift.com/api/accounts_mgmt/v1/access_token | sed 's/"/\\"/g')
+
+CLUSTER_ID=$(curlcurl -sSf -X POST -H "Content-Type: application/json" -H "Authorization: Bearer ${OCM_TOKEN}" \
+    -d "{\"name\":\"eval-testcluster-${UNIQUE_ID}\",\"control_plane_count\":1,\"openshift_version\":\"4.18.22\",\"pull_secret\":\"${PULL_SECRET}\",\"base_dns_domain\":\"test.local\"}" \
+    "${ASSISTED_SERVICE_URL}/clusters" | jq '.id')
+
+#CLUSTER_ID is already quoted
+curl curl -sSf  -X POST -H "Content-Type: application/json" -H "Authorization: Bearer ${OCM_TOKEN}" \
+    -d "{\"name\":\"eval-testcluster-${UNIQUE_ID}_infra-env\",\"pull_secret\":\"${PULL_SECRET}\",\"cluster_id\":${CLUSTER_ID},\"openshift_version\":\"4.18.22\"}" \
```

Comment:
Will this `UNIQUE_ID` be set when running the eval tests locally? Can we change this setup stuff such that it can be used when running the eval tests locally as well?